src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1450,12 +1450,24 @@
   </Node>
 
   <Node Name="BoundSubPropertyPattern" Base="BoundNode">
-    <Field Name="Type" Type="TypeSymbol" Null="disallow"/>
-    <Field Name="Property" Type="Symbol" Null="allow"/>
-    <Field Name="ResultKind" Type="LookupResultKind"/>
+    <!-- Member is either a BoundPropertyPatternMember or an error expression -->
+    <Field Name="Member" Type="BoundExpression" Null="disallow"/>
     <Field Name="Pattern" Type="BoundPattern" Null="disallow"/>
   </Node>
 
+  <Node Name="BoundPropertyPatternMember" Base="BoundExpression">
```

Comment:
There would be nothing to do in that override. The implicit receiver for this property invocation is a copy of the receiver held by the enclosing node.